dulwich/tests/test_porcelain.py

```diff
@@ -53,6 +55,34 @@
     )
 
 
+def create_dir_in_wd(repo, dir_path):
```

Comment:
Please don't add lots of helper functions; it makes the tests hard to read and it doesn't actually make the code any shorter. This particular function can just be replaced with "os.mkdir(dir_path)"